handler/commentHandler.go

```diff
@@ -426,6 +433,8 @@ func parse(body string, commandTriggers []string) *types.CommentAction {
 			commandTrigger + "remove milestone: ": removeMilestoneConstant,
 			commandTrigger + "set reviewer: ":     assignReviewerConstant,
 			commandTrigger + "clear reviewer: ":   unassignReviewerConstant,
+			commandTrigger + "message: ":          messageConstant,
+			commandTrigger + "msg: ":              messageAliasConstant,
```

Comment:
We do need the alias here, but they can both point at the same const.

Reply:
Both pointing at the same constant 👍